components/onboarding-modal/OnboardingContentBox.js

```diff
@@ -0,0 +1,165 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { Flex, Box } from '@rebass/grid';
+
+import Container from '../../components/Container';
+import { H1, P } from '../../components/Text';
+import StyledInputField from '../../components/StyledInputField';
+import StyledInputGroup from '../../components/StyledInputGroup';
+import StyledHr from '../../components/StyledHr';
+import CollectivePickerAsync from '../../components/CollectivePickerAsync';
+import OnboardingProfileCard from './OnboardingProfileCard';
+
+class OnboardingContentBox extends React.Component {
+  static propTypes = {
+    step: PropTypes.number,
+    collective: PropTypes.object,
+    LoggedInUser: PropTypes.object,
+    addAdmins: PropTypes.func,
+    addContact: PropTypes.func,
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      admins: [],
+    };
+  }
+
+  componentDidMount() {
+    const member = this.props.LoggedInUser.memberOf.filter(member => member.collective.id === this.props.collective.id);
+    this.setState({
+      admins: [{ role: 'ADMIN', member: this.props.LoggedInUser.collective, id: member[0].id }],
+    });
+  }
+
+  render() {
+    const { step, collective, LoggedInUser, addAdmins, addContact } = this.props;
+    const { admins } = this.state;
+
+    return (
+      <Container display="flex" flexDirection="column" width="80%" alignItems="center">
+        {step === 0 && (
+          <Flex maxWidth={['336px']}>
+            <H1
+              fontSize={['H5']}
+              lineHeight={['H5']}
+              fontWeight="bold"
+              color="black.900"
+              textAlign="center"
+              mb={[2, 4]}
+              mx={[2, null]}
+            >
+              The {collective.name} Collective has been created! 🎉
+            </H1>
+          </Flex>
+        )}
+        {step === 1 && (
+          <Fragment>
+            <Flex maxWidth={['336px']}>
+              <H1 fontSize={['H5']} lineHeight={['H5']} fontWeight="bold" color="black.900" textAlign="center" mb={4}>
+                Add administrators
+              </H1>
+            </Flex>
+            <Flex px={3} width="100%">
+              <P my={2} fontSize="Caption" textTransform="uppercase" color="black.700">
+                Administrators
+              </P>
+              <Flex flexGrow={1} alignItems="center">
+                <StyledHr width="100%" ml={2} />
+              </Flex>
+            </Flex>
+            {admins.length > 0 && (
+              <Flex px={3} width="100%" flexWrap="wrap">
+                {admins.map(admin => (
+                  <OnboardingProfileCard
+                    key={admin.member.id}
+                    user={admin.member}
+                    adminUser={LoggedInUser.collective}
```

Comment:
**[:sparkles: **Enhancement**]** If we're passing collectives, then the props should be `collective` and `adminCollective`. If the prop is called user, I'm tempted to poss `admin.user` and `LoggedInUser` there.

---

<details>
<summary>More details about this feedback</summary>
<br/>

This feedback categorization is inspired by [Netlify's Feedback Ladders system](https://www.netlify.com/blog/2020/03/05/feedback-ladders-how-we-encode-code-reviews-at-netlify). It includes 5 levels of feedback:

:rotating_light: **Critical blocker**
This feedback blocks all related work, and requires immediate action. 

:no_entry_sign: **Blocker**
This feedback should block the work from moving forward or being approved, but it does not necessarily require immediate action from the team.

:hammer_and_wrench: **Important**
This feedback should not block the work from moving forward or being approved, but it does require future action from the team. This generally means the feedback is not required for the MVP or initial draft of this work, but should be addressed at some point and we should ideally create an issue to track it.

:sparkles: **Enhancement**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they agree with it.

:ice_cream: **Details & personal preferences**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they deem it useful. This is also the category for everything that relates to personal preferences or code style.

</details>

Reply:
A bit of explanation for this:

* `admins` is the list of `User` objects stored when we add a User from `CollectivePickerAsync`, so `admin.member` is this object, and it allows us to access the name, avatar src, etc. I could change `admin.member` to `admin.user` but it would require changing the GraphQL stuff as well, which isn't a huge deal, but unless it's a big issue, I might leave it til the end.

Re: `adminUser` this is actually no longer needed so thanks for bringing it to my attention!

Reply:
`admin.member` is a collective, not a user: https://github.com/opencollective/opencollective-api/blob/cc748aa63a36b19d1a0b9914037e94dd84307d85/server/graphql/v1/types.js#L360. 

The user is stored in `admin.member.createdByUser` but I don't think you need it.

Reply:
I see! I will change this in my next PR then.